testsuite/MDAnalysisTests/analysis/test_encore.py

```diff
@@ -128,7 +128,7 @@ def test_triangular_matrix():
 
         multiplied_triangular_matrix_2 = triangular_matrix_2 * scalar
         assert_equal(multiplied_triangular_matrix_2[0,1], expected_value * scalar,
-			err_msg="Error in TriangularMatrix: multiplication by scalar gave\
+            err_msg="Error in TriangularMatrix: multiplication by scalar gave\
```

Comment:
please remove the trailing `\` and align with the opening brackets in the previous line